study.md

```diff
@@ -20,19 +20,35 @@ There is no requirement to follow any links from these pages.
 
 After doing all the required readings please tell us anything you were confused about.
 
+
+I am confused about “hoisting” and what is getting hoisted? that went over my head, I read it a few times and couldn’t figure it out.
+As far as some of the readings I am confused when new “terms” seem to be included in functions without them being declared: as an example :
+
+var friends = ["Mike", "Stacy", "Andy", "Rick”];
+ friends.forEach(function(eachName, index){
+console.log(index+1+". "+eachName);
+});
+How did the eachName and index get defined?
```

Comment:
```eachName``` and ```index``` are the arguments of the callback function being passed into ```forEach```.  ```forEach``` is an iterator method on arrays.  You'll be learning about array methods this coming week.  Here's some more information on the ```forEach``` method.  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach